tests/e2e/DocumentTest.spec.js

```diff
@@ -1,35 +1,39 @@
 const config = require('../../nightwatch.json');
+require('dotenv').config();
 
 module.exports = {
-  'User should be able to see all documents afetr login': (browser) => {
+  'User should be able to see all documents after login': (browser) => {
     browser
       .resizeWindow(1280, 800)
       .url('http://localhost:8080/checkin')
       .waitForElementVisible('body', 5000)
-      .setValue('input[name=email]', 'hello2@hopez.com')
-      .setValue('input[name=password]', 'password')
+      .setValue('input[name=email]', process.env.ADMIN_EMAIL)
+      .setValue('input[name=password]', process.env.ADMIN_PASSWORD)
       .click('button[id="hopez-save"]')
-      .pause(5000)
+      .waitForElementVisible('.toast', 3000)
+      .assert.containsText('.toast', 'Successfully logged in')
       .assert.urlEquals('http://localhost:8080/documents')
-      .pause(1000);
+      .waitForElementVisible('.card.small', 3000)
+      .assert.elementPresent('.card.small');
   },
   'User should be able to search for documents': (browser) => {
     browser
     .resizeWindow(1280, 800)
       .waitForElementVisible('#search-input', 3000)
       .click('#search-input')
-      .setValue('input[id=search-input]', 'Test')
-      .pause(2000)
-      .waitForElementVisible('#search-btn', 1000)
-      .click('#search-btn')
-      .pause(1000);
+      .setValue('input[id=search-input]', 'hello')
+      .pause(5000)
+      .waitForElementVisible('.card.small', 3000)
+      .assert.elementPresent('.card.small')
+      .waitForElementVisible('#search-btn', 3000)
+      .click('#search-btn');
```

Comment:
It seems to me like this test is testing that we can click on a search button. as opposed to testing for what the description says it's testing for.